frb_spectral_search/acf_func.py

```diff
@@ -47,8 +47,8 @@ def calc_fdcs(all_acf, bands):
     plt.ylabel("f_dc")
     plt.legend()
     plt.show()
-    
-def acf_naive2(spectra, n, plot_range = None, plot=True):   
+
+def acf_naive2(spectra, n, plot_range = None, plot=True):
     spec_bigsub = gauss_subtract_wo_and_replace_zero(spectra, 600)[150:-149] #double removing 150?
```

Comment:
What does `600` mean here? And what do [150:-149] mean here? Looks like an old smoothing method.